yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -774,3 +777,133 @@ func (yb *TargetYugabyteDB) recordEntryInDB(tx pgx.Tx, batch Batch, rowsAffected
 	}
 	return nil
 }
+
+func (yb *TargetYugabyteDB) GetDebeziumValueConverterSuite(isStreamingMode bool) map[string]func(string) (string, error) {
+	converterValueSuite := make(map[string]func(string) (string, error))
+	converterValueSuite["io.debezium.time.Date"] = func(columnValue string) (string, error) {
+		epochDays, err := strconv.ParseUint(columnValue, 10, 64)
+		fmt.Printf("epochDays: %v", epochDays)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch seconds: %v", err)
+		}
+		epochSecs := epochDays * 24 * 60 * 60
+		return time.Unix(int64(epochSecs), 0).Local().Format(time.DateOnly), nil
+	}
+	converterValueSuite["io.debezium.time.Timestamp"] = func(columnValue string) (string, error) {
+		epochMilliSecs, err := strconv.ParseInt(columnValue, 10, 64)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch milliseconds: %v", err)
+		}
+		epochSecs := epochMilliSecs / 1000
+		return time.Unix(epochSecs, 0).Local().Format(time.DateTime), nil
+	}
+	converterValueSuite["io.debezium.time.MicroTimestamp"] = func(columnValue string) (string, error) {
+		epochMicroSecs, err := strconv.ParseInt(columnValue, 10, 64)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch microseconds: %v", err)
+		}
+		epochSeconds := epochMicroSecs / 1000000
+		epochNanos := (epochMicroSecs % 1000000) * 1000
+		microTimeStamp, err := time.Parse(time.RFC3339Nano, time.Unix(epochSeconds, epochNanos).Local().Format(time.RFC3339Nano))
+		if err != nil {
+			return columnValue, err
+		}
+		return strings.TrimSuffix(microTimeStamp.String(), " +0000 UTC"), nil
+	}
+	converterValueSuite["io.debezium.time.NanoTimestamp"] = func(columnValue string) (string, error) {
+		epochNanoSecs, err := strconv.ParseInt(columnValue, 10, 64)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch nanoseconds: %v", err)
+		}
+		epochSeconds := epochNanoSecs / 1000000000
+		epochNanos := epochNanoSecs % 1000000000
+		nanoTimeStamp, err := time.Parse(time.RFC3339Nano, time.Unix(epochSeconds, epochNanos).Local().Format(time.RFC3339Nano))
```

Comment:
In all the time formating, pls make sure it's UTC.

Reply:
this `time` package, by default, it gives UTC format only unless we specify. So all the time formatting here is in UTC.